services/storage-users/pkg/config/config.go

```diff
@@ -24,10 +24,13 @@ type Config struct {
 	SkipUserGroupsInToken   bool `yaml:"skip_user_groups_in_token" env:"STORAGE_USERS_SKIP_USER_GROUPS_IN_TOKEN" desc:"Disables the loading of user's group memberships from the reva access token."`
 	GracefulShutdownTimeout int  `yaml:"graceful_shutdown_timeout" env:"STORAGE_USERS_GRACEFUL_SHUTDOWN_TIMEOUT" desc:"The number of seconds to wait for the 'storage-users' service to shutdown cleanly before exiting with an error that gets logged. Note: This setting is only applicable when running the 'storage-users' service as a standalone service. See the text description for more details."`
 
-	Driver            string            `yaml:"driver" env:"STORAGE_USERS_DRIVER" desc:"The storage driver which should be used by the service. Defaults to 'ocis', Supported values are: 'ocis', 's3ng' and 'owncloudsql'. The 'ocis' driver stores all data (blob and meta data) in an POSIX compliant volume. The 's3ng' driver stores metadata in a POSIX compliant volume and uploads blobs to the s3 bucket."`
-	Drivers           Drivers           `yaml:"drivers"`
-	DataServerURL     string            `yaml:"data_server_url" env:"STORAGE_USERS_DATA_SERVER_URL" desc:"URL of the data server, needs to be reachable by the data gateway provided by the frontend service or the user if directly exposed."`
-	DataGatewayURL    string            `yaml:"data_gateway_url" env:"STORAGE_USERS_DATA_GATEWAY_URL" desc:"URL of the data gateway server"`
+	Driver         string  `yaml:"driver" env:"STORAGE_USERS_DRIVER" desc:"The storage driver which should be used by the service. Defaults to 'ocis', Supported values are: 'ocis', 's3ng' and 'owncloudsql'. The 'ocis' driver stores all data (blob and meta data) in an POSIX compliant volume. The 's3ng' driver stores metadata in a POSIX compliant volume and uploads blobs to the s3 bucket."`
+	Drivers        Drivers `yaml:"drivers"`
+	DataServerURL  string  `yaml:"data_server_url" env:"STORAGE_USERS_DATA_SERVER_URL" desc:"URL of the data server, needs to be reachable by the data gateway provided by the frontend service or the user if directly exposed."`
+	DataGatewayURL string  `yaml:"data_gateway_url" env:"STORAGE_USERS_DATA_GATEWAY_URL" desc:"URL of the data gateway server"`
+
+	GatewayGRPCAddr string `yaml:"gateway_addr" env:"GATEWAY_GRPC_ADDR" desc:"The bind address of the GRPC service."`
```

Comment:
`GATEWAY_GRPC_ADDR`
You cant add that envvar to the `storage-users` service --> we either need a `STORAGE_USERS_GATEWAY_GRPC_ADDR` or better, in addition a global one `OCIS_GATEWAY_GRPC_ADDR`.

```
grep -rn GATEWAY_GRPC_ADDR

web/pkg/config/config.go:27:	        
GatewayAddress string `yaml:"gateway_addr" env:"WEB_GATEWAY_GRPC_ADDR" 
desc:"The bind address of the GRPC service."`

gateway/pkg/config/config.go:73:	     
     Addr      string `yaml:"addr" env:"GATEWAY_GRPC_ADDR" 
desc:"The bind address of the GRPC service."`

settings/pkg/config/config.go:49:	
GatewayAddress string `yaml:"gateway_addr" env:"STORAGE_GATEWAY_GRPC_ADDR" 
desc:"GRPC address of the STORAGE-SYSTEM service."`
```

Sidenote: `GATEWAY_GRPC_ADDR` in the gateway service is naming wise wrong.
It should be: `GATEWAY_GATEWAY_GRPC_ADDR`...

Reply:
I want to use exists one, that is reva gateway`GATEWAY_GRPC_ADDR: 0.0.0.0:9142`
Looks like I need to change the variable description...